frontend/src/components/Sidebar/DistrictMap.tsx

```diff
@@ -275,12 +276,28 @@ export default function DistrictMap(): JSX.Element {
               legendRef.current = legend;
             }}
             min={0}
-            max={legend.isNormalized ? aggregatedMax : legend.steps[legend.steps.length - 1].value}
+            // used math.round to convert the number in integers and used +1 to avoid having legend max value as 0
+            max={
+              legend.isNormalized
+                ? Math.round(aggregatedMax) + 1
+                : Math.round(legend.steps[legend.steps.length - 1].value) + 1
+            }
             displayText={true}
             id={'legend'}
           />
         </Grid>
         <Grid item container justifyContent='center' xs={1}>
+          <ZoomOutMapIcon
+            id='home'
+            color='primary'
+            fontSize='large'
+            // if the chart is zoomed then, it will bring the chart to its default state, plus change the values to Germany wide.
+
+            onClick={() => {
+              dispatch(selectDistrict({ags: '00000', name: t('germany'), type: ''}));
```

Comment:
why are we chanign the selected district too on the zoom out button?
To me it seems more intuitive if the Zoom out button just zooms and pans the map back into its default position.

Or did we have another reason for this?
```suggestion
```